check/all

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env bash
+
+################################################################################
+# Runs multiple checks.
+#
+# The default set of checks most closely matches the tests run during
+# a continuous integration test, however these do not match perfectly
+# since those tests run on different architectures and environments.
+#
+# Defaults to the following tests:
+#   pylint
+#   mypy
+#   format-incremental
+#   pytest-and-incremental-coverage
+#   doctest
+#
+# Usage:
+#     check/all [--only-changed-files] [--apply-format]
+#
+# If --only-change-files is specified, pytest-changed-files will be run
+# instead of pytest-and-incremental-coverage.
+#
+# If --apply-format-changes is specified the --apply flag will be passed
+# to format-incremental to apply the format changes suggested by the
+# formatter.
+################################################################################
+
+# Get the working directory to the repo root.
+cd "$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+cd "$(git rev-parse --show-toplevel)"
+
+# Parse arguments.
+apply=0
+only_changed=0
+for arg in $@; do
```

Comment:
We need to allow the user to specify the revision the compare against. If there's one non-dash argument, it should be forwarded into the incremental scripts.

Reply:
Done.  I never use anything besides the default :)